xAPI-Communication.md

```diff
@@ -1466,7 +1464,7 @@ If the header precondition in any of the POST or DELETE request cases above fail
 If a PUT request is received without either header for a resource that already exists, the LRS:
 
 * MUST return HTTP status 409 "Conflict".
-* MUST return a response explaining that the consumer SHOULD
+* MUST return a response explaining that the Client SHOULD
```

Comment:
As you can tell, I'm a bit gunshy on being too specific on some of these.  Thanks for clarifying!